docs/queryworkers/akamai/delete-edge-workers.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 20
+title: Delete Edge Workers
+---
+
+Delete an edge worker.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Functions**.
+1. Click the stacked dots next to the edge worker you want to delete.
+1. Click **Delete Edge Worker**.
+1. Confirm your choice.
+    Macrometa permanently deletes the edge worker. You can re-create the edge worker, but you cannot undo the deletion.
```

Comment:
Check your linter. Looks like there is one space too many in front of this paragraph.